ui/src/chatbot.tsx

```diff
@@ -101,7 +96,9 @@ export const XChatbot = ({ model }: { model: Chatbot }) => {
       setUserInput('')
     }
 
-  React.useEffect(() => { if (msgContainerRef.current) msgContainerRef.current.scrollTop = msgContainerRef.current.scrollHeight }, [msgs])
+  React.useEffect(() => {
+    if (msgContainerRef.current) msgContainerRef.current.scrollTo({ top: msgContainerRef.current.scrollHeight, behavior: 'smooth' })
```

Comment:
ScrollTo is only [supported](https://caniuse.com/?search=scrollTo) on Safari (fully) & Edge from 2020, let's add a fallback to the previous `scrollTop` in case it's not available.

Reply:
On Safari only `smooth` behavior is not working in 2017-2020 versions but otherwise it works the same as setting the `scrollTop` (without the animation). That's why we previously agreed using it in the image popup (lightbox). But I'll add a fallback for older versions of MS Edge.